src/App.js

```diff
@@ -1,29 +1,21 @@
 import Button from './components/Button';
-const handelClick = () => {
-  console.log("button clicked!");
-};
+import TextField from './components/TextField';
+import MuiDrawer from './components/MuiDrawer';
+import HomePage  from "./components/HomePage";
```

Comment:
A few imports are unused. Consider installing the `ESLint` extension in VSCode to receive warnings about issues like this:
![49778c9af4dbc4190e8d83641f89d266](https://github.com/user-attachments/assets/976d8bd1-d286-454c-88c9-12ad1532fcb7)